updates/ryan_verbesserung.sql

```diff
@@ -0,0 +1,13 @@
+
+INSERT INTO `neg`.`datenbank_texte` (`Formular`, `Textfeld`, `de`, `gb`, `fr`, `la`) VALUES ('einzelbeleg', 'ZusatznamenkommentarPopup', 'Für Belegform %s wurde Zusatznamenkommentar %s gefunden. NamenkommentarID: %s, EinzelbelegIDs: %s', 'For form of reference %s, additional name comment %s was found. Commentary(name)ID %s, Single referenceIDs: %s', 'Pour la forme de référence %s, un commentaire sur le nom supplémentaire %s a été trouvé. ID du commentaire %s, références individuelles IDs : %s', 'Pro forma testimonii %s, commentarium additicius nominis %s repertum est. ID commentarii %s, testimonium individuorum IDs: %s');
+
+
+INSERT INTO `neg`.`datenbank_texte` (`Formular`, `Textfeld`, `de`, `gb`, `fr`, `la`) VALUES ('einzelbeleg', 'ChoicePopupZusatznamenkommentar', 'Soll dieser Zusatznamenkommentar hier übernommen werden?', 'Should this additional name comment be adopted here?', 'Ce commentaire sur le nom supplémentaire doit-il être adopté ici ?', 'Debetne hoc commentarium nominis additicius hic assumptum?');
+
+INSERT INTO `neg`.`datenbank_texte` (`Formular`, `Textfeld`, `de`, `gb`, `fr`, `la`) VALUES ('einzelbeleg', 'ChoiceCleanDataPopup', 'Bitte bereinigen Sie die Daten!', 'Please clean up the data!', 'Veuillez nettoyer les données !', 'Quaeso data purganda sunt!');
```

Comment:
Please think about a naming convention and stick to it:
- LemmaPopup
- ChoicePopupLemma
- ChoiceCleanDataPopup
...

Reply:
done